homeassistant/components/config/zwave.py

```diff
@@ -52,7 +52,7 @@ def get(self, request, lines):
             else:
                 loglines = list(deque(data, lines))
 
-        return self.json(loglines)
+        return self.json('\n'.join(loglines))
```

Comment:
Don't return `json`. Instead, return just plain text. 

```
# At top of file
from aiohttp.web import Response

return Response(text='\n'.join(loglines))
```

Reply:
Also means that you no longer have to convert your deque to a list.

Reply:
This method gives pylint: Method could be a function (no-self-use) Is that safe to ignore?